securedrop_client/db.py

```diff
@@ -544,7 +544,7 @@ def __repr__(self) -> str:
 
     @property
     def deleted(self) -> bool:
-        return True if self.uuid == "deleted" else False
+        return True if self.username == "deleted" else False
```

Comment:
nit: This can be simplified to just `return self.username == "deleted"`

Reply:
explicitly returning false or true seems clear. either way. i think i'll just leave it